src/test/java/se/sundsvall/incident/service/IncidentServiceTest.java

```diff
@@ -26,173 +27,209 @@
+	void updateIncidentStatusNotFoundTest() {
+		var entity = createIncidentEntity();
+		when(mockIncidentRepository.findById(any())).thenReturn(Optional.empty());
 
-		when(incidentRepository.findAll(any(PageRequest.class))).thenReturn(pagey);
+		assertThatThrownBy(() -> incidentService.updateIncidentStatus(entity.getIncidentId(), anyInt()))
+			.isInstanceOf(Problem.class)
+			.hasMessageContaining("Not Found: Incident with id: ");
 
-		final var response = incidentService.getIncidents(1, 4);
+		assertThat(entity.getStatus()).isEqualTo(Status.INSKICKAT);
+		verify(mockIncidentRepository, times(1)).findById(any());
+		verify(mockIncidentRepository, never()).save(entity);
+	}
+
+	@Test
+	void updateIncidentFeedbackTest() {
+		var entity = createIncidentEntity();
+		when(mockIncidentRepository.findById(any())).thenReturn(Optional.ofNullable(entity));
 
-		assertThat(response)
-			.isNotNull()
-			.hasSize(4);
+		incidentService.updateIncidentFeedback(entity.getIncidentId(), "Feedback!!");
+
+		assertThat(entity.getFeedback()).isEqualTo("Feedback!!");
+		verify(mockIncidentRepository, times(1)).findById(any());
+		verify(mockIncidentRepository, times(1)).save(entity);
+		verifyNoMoreInteractions(mockIncidentRepository);
+	}
+
+	@Test
+	void updateIncidentFeedbackNotFoundTest() {
+		var entity = createIncidentEntity();
+		when(mockIncidentRepository.findById(any())).thenReturn(Optional.empty());
+
+		assertThatThrownBy(() -> incidentService.updateIncidentFeedback(entity.getIncidentId(), anyString()))
+			.isInstanceOf(Problem.class)
+			.hasMessageContaining("Not Found: Incident with id: ");
+
+		verify(mockIncidentRepository, times(1)).findById(any());
+		verify(mockIncidentRepository, never()).save(entity);
+	}
+
+	@Test
+	void sendNotification_LIVBAT_Test() throws JsonProcessingException {
+		when(mockLifeBuoyIntegration.sendLifeBuoy(any())).thenReturn("nothing");
+		var entity = createIncidentEntity();
+		var category = createCategoryEntity();
+		category.setTitle("LIVBAT");
+		entity.setCategory(category);
+
+		incidentService.sendNotification(entity);
+
+		verify(mockLifeBuoyIntegration, times(1)).sendLifeBuoy(any());
 	}
 
 	@Test
-	void updateIncidentFeedback() {
-		final var incidentEntity = buildIncidentEntity(Category.FELPARKERAD_BIL);
-		when(incidentRepository.findById(anyString())).thenReturn(Optional.of(incidentEntity));
+	void sendNotification_VATTENMATARE_Test() {
+		when(mockMessagingIntegration.sendMSVAEmail(any())).thenReturn(Optional.of(new MessageResult()));
+		var entity = createIncidentEntity();
+		var category = createCategoryEntity();
+		category.setTitle("VATTENMATARE");
+		entity.setCategory(category);
 
-		incidentService.updateIncidentFeedback(INCIDENTID, "someFeedback");
-		verify(incidentRepository, times(1)).findById(any());
-		verify(incidentRepository, times(1)).save(any());
+		incidentService.sendNotification(entity);
+
+		verify(mockMessagingIntegration, times(1)).sendMSVAEmail(any());
 	}
 
```

Comment:
Add test for sending notification for other than vattenmätare and livbåt to verify default behavior in service (using messagingIntegration.sendEmail)